test/ng/filter/filterSpec.js

```diff
@@ -428,6 +428,32 @@ describe('Filter: filter', function() {
   });
 
 
+  it('should not throw an error if property is null when comparing object', function() {
+      var items = [
+          { office:1, people: {name:'john'}},
+          { office:2, people: {name:'jane'}},
+          { office:3, people: null}
+      ];
+      var f = { };
+      expect(filter(items, f).length).toBe(3);
+
+      f = { people:null };
+      expect(filter(items, f).length).toBe(3);
```

Comment:
Hm...this test shouldn't pass (the length should be 1). But it does pass since version 1.3.6.
This made me realize that there is a similar bug with `expectedType` (see above).